vision/unit_tests/test_vector_utils.py

```diff
@@ -0,0 +1,423 @@
+"""
+Testing vision.deskew.vector_utils.py
+"""
+
+import unittest
+import numpy as np
+from scipy.spatial.transform import Rotation
+
+import vision.deskew.vector_utils as vu
+from vision.common.constants import Vector, Point, SENSOR_HEIGHT, SENSOR_WIDTH, ROTATION_OFFSET
+
+
+Test_IHAT: Vector = np.array([1, 0, 0], dtype=np.float64)
+
+# NOTE: These tests are ordered with the helper functions first before the
+#       other functions that use them
+
+
+class TestIHAT(unittest.TestCase):
+    """
+    Unit test for the IHAT constant
```

Comment:
Adding docstrings for classes and functions according to [Multirotor docs](https://missourimrr.github.io/docs/python/documenting-code/docstrings/#docstrings).

Reply:
Would I need a "Raises" section in these functions? As a lot of the headings seemingly can be omitted as there are no parameters to pass in nor do they return anything? Also I'm guessing just something similar to the other unit test that's already present?

Reply:
Correct, no need for any Parameters or Returns sections. Most likely, none of the other sections will be needed either.

Reply:
How does the updated docstrings look for the latest commit?